istio.deps

```diff
@@ -4,7 +4,7 @@
 		"name": "PROXY_REPO_SHA",
 		"repoName": "proxy",
 		"file": "",
-		"lastStableSHA": "542b4fda35046fc9dad5014dbe81f3d5969e63f7"
```

Comment:
I would do this in a separate PR (before this one) to avoid any complication if we need to rollback.

Reply:
Created https://github.com/istio/istio/pull/15342